src/index.html

```diff
@@ -7,11 +7,158 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
+    <title>Personal information</title>
     <link rel="stylesheet" href="./style.css">
+    <script type="text/javascript" src="./main.js"></script>
+
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+
+    <form tabindex="1">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname">Surname:</label>
+        <input type="text"
+        name="usersurname"
+        autocomplete="off"
+        ><br>
+
+        <label for="name">Name:</label>
```

Comment:
Either add `id` attribute to <input> tag
or just wrap `<input>` into a `<label>` (this is a better variant and you don't need to use `for`/`id` in this case)